prototype1/prototype_v1_JC_example.yml

```diff
@@ -0,0 +1,29 @@
+author:
+  -
+    string-name:
+    surname: Colomb
+    given-names: Julien
+    email: julien.colomb@fu-berlin.de
+    PID:
+      type: orcid-id
+      id: https://orcid.org/0000-0002-3127-5520
```

Comment:
Alternative suggestion for the IDs: Instead of defining an arbitrary top-level-property, and nesting the properties `type` and `id` below that, we could capture the JATS XML `contrib-id` with corresponding `contrib-type` with top-level properties instead. That is (e.g.):

```yaml
author:
    surname: Mustermann
    given-names: Max
    orcid: 0000-0002-3127-5520
    academia-id: https://academia.edu/link/to/profile # Just an arbitrary example
    scopus-id: 7007156898 # Taken from https://jats.nlm.nih.gov/archiving/tag-library/1.1/element/contrib-id.html
```

This might work better, as it …

1. … reduces the mental load for authors writing the frontmatter
2. … contains the same amount of information
3. … is simpler and easier for the parsers

Reply:
I like this approach. Maybe something similar could be done with `attributes`?

Reply:
What are you referring to with attributes?

In general, I would say where we have a principally closed list of attribute values, we should include those in the property name (there are not endless ways to identify authors or institutions), whereas for arbitrary property values we should use this approach to create a "top level" property!

Reply:
I was refering to the `attributes` component of the author list: https://github.com/jam-schema/jams/blob/b23338016ccb6e0527b1f90c5e2ca3c25589fccf/prototype1/prototype_v1_JC_example.yml#L15

Reply:
Let's discuss this in #20.

Reply:
This was rejected by the group